server/views/dashboard/index.njk

```diff
@@ -50,6 +50,17 @@
         </div>
       {% endif %}
 
+      {% if 'ROLE_ACP_HSP' in user.roles %}
```

Comment:
Probably worth adding checking as well for the `HSP_ENABLED` flag. Incase roles get assigned before we enable it.